plugins/modules/container_distribution.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# copyright (c) 2021, Mark Goddard
+# copyright (c) 2020, Jacob Floyd
```

Comment:
Is this leftover copy and paste?
I don't think, we need it for the boiler plate. All the names will be in the collections galaxy.yaml file anyway.